sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/ItemPage.java

```diff
@@ -30,8 +31,10 @@
     private String continuationToken;
 
     @Override
-    public List<T> getItems() {
-        return items;
+    public IterableStream<T> getElements() {
+        return items == null
```

Comment:
Same question about returning a null instead of an empty collection type.

Reply:
I agree with Alan

Reply:
yes, updated to return empty iterable-stream.